packages/core/src/utilities/elementFromString.ts

```diff
@@ -20,7 +20,5 @@ export function elementFromString(value: string): HTMLElement {
 
   const html = new window.DOMParser().parseFromString(wrappedValue, 'text/html').body
 
-  removeWhitespaces(html)
```

Comment:
I'll check this issue again on another branch afterwards.

Reply:
Yeah, only removed the duplicate function call here. Shouldn't have changed anything with the other issue.